discovery-provider/src/queries/get_health.py

```diff
@@ -314,6 +314,9 @@ def get_health(args: GetHealthArgs, use_redis_cache: bool = True) -> Tuple[Dict,
     url = shared_config["discprov"]["url"]
     final_poa_block = helpers.get_final_poa_block(shared_config)
 
+    auto_upgrade_enabled = True if os.getenv("audius_auto_upgrade_enabled") == 'true' else False
+    database_is_localhost = os.getenv("audius_db_url").find('localhost') > 0
```

Comment:
i think same same goes here as for the content node check. we should check if exact match for this string or contains localhost https://github.com/AudiusProject/audius-docker-compose/blob/f4eebde6097e9431304fbaf68f4b878ef0b73586/discovery-provider/prod.env#L5